templates/423.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+{% set active_page = "support" %}
+
+{% block content %}
+<div class="container">
+    <div class="row" style="margin-top:50px;">
+        <div class="col-md-12">
+            <h1><strong>Unable to Access Project</strong></h1>
+            <h3>Please confirm your project's password settings and assigned workers.</h3>
+
+            <h4><strong>Public GIGwork:</strong></h4>
+            <p>Ask a project admin to ensure that the project has a password set OR that you are assigned to the
+                project.</p>
+
+            <h4><strong>Private GIGwork:</strong></h4>
+            <p>Ask a project admin to ensure that you are assigned to the project.</p>
+            <hr>
```

Comment:
This can be consolidated on the UI into single statement upon passing to the type of server from `settings.SERVER_TYPE`
```
<h4><strong>{{ server_type }}</strong></h4>
{% if server_type.startswith('Public') %}
      <p>Ask a project admin to ensure that the project has a password set OR that you are assigned to the
                project.</p>
{% else %}
      <p>Ask a project admin to ensure that you are assigned to the project.</p>
{% endif %}
```